android/testng-examples/browserstack.yml

```diff
@@ -0,0 +1,77 @@
+# =============================
+# Set BrowserStack Credentials
+# =============================
+# Add your BrowserStack userName and acccessKey here or set BROWSERSTACK_USERNAME and
+# BROWSERSTACK_ACCESS_KEY as env variables
+userName: BROWSERSTACK_USERNAME
+accessKey: BROWSERSTACK_ACCESS_KEY
+
+# ======================
+# Organizing your tests
+# ======================
+# Use `projectName`, `buildName`, `name` capabilities to organise your tests
+# `name` is the name of your test sessions and is automatically picked from your
+# test name and doesn't need to be set manually when using BrowserStack SDK
+# `buildName` is used to name your CI/CD job or the execution of your test suite.
+# Ensure you add a dynamic identifier, like an incremental build number from your
+# CI/CD or timestamp at the end of every build; otherwise tests from different
+# executions will be grouped together on BrowserStack
+buildName: browserstack-build
```

Comment:
```suggestion
buildName: browserstack-build-1
```

Reply:
We're auto-incrementing counter, So not needed.